ui/shared/LabelValueRow.qml

```diff
@@ -0,0 +1,32 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtQuick.Layouts 1.13
+import "../imports"
+
+Item {
+    id: itmFrom
+    property alias label: txtLabel.text
+    property alias value: itmValue.children
+    Layout.alignment: Qt.AlignTop | Qt.AlignLeft
+    Layout.preferredWidth: parent.width
+    width: parent.width
+    height: 52
+
+    StyledText {
+        id: txtLabel
+        font.pixelSize: 15
+        height: parent.height
+        anchors.left: parent.left
+        anchors.leftMargin: Style.current.padding
+        anchors.verticalCenter: parent.verticalCenter
+        verticalAlignment: Text.AlignVCenter
+        width: 105
+    }
+    Item {
+        id: itmValue
+        anchors.right: parent.right
+        anchors.verticalCenter: parent.verticalCenter
+        height: parent.height
+        width: parent.width - (Style.current.padding + txtLabel.width)
```

Comment:
Again here, you could instead just anchor right on the parent and left on the `txtLabel` with your own padding, that way you don't have to make calculations.
This one is way more simple, so it's not as big of a deal, but it's also way simpler to implement 😛